algorithms/kernel/linear_model/linear_model_predict_batch_fpt.cpp

```diff
@@ -42,7 +43,18 @@ DAAL_EXPORT Status Result::allocate(const daal::algorithms::Input * input, const
     size_t nVectors            = in->get(data)->getNumberOfRows();
     size_t nDependentVariables = in->get(model)->getNumberOfResponses();
     Status st;
-    set(prediction, HomogenNumericTable<algorithmFPType>::create(nDependentVariables, nVectors, NumericTable::doAllocate, &st));
+
+    auto & context    = services::Environment::getInstance()->getDefaultExecutionContext();
```

Comment:
Are these changes in the master?

Reply:
Yes, it's cherry-pick of commit from master